Login_SpotifyUI/Login_SpotifyUI/Controller/SignUpViewController.swift

```diff
@@ -110,24 +125,27 @@ class SignUpViewController: UIViewController {
         animateView(viewToAnimate: sender)
         
         guard let identification = signUpView.identificationTextField.text else { return }
+        guard let nickName = signUpView.nickNameTextField.text else { return }
```

Comment:
nickname은 단어 그 자체로 별칭이라는 뜻이므로 중간에 camelcase로 표시할 필요가 없습니다~!
```suggestion
        guard let nickname = signUpView.nickNameTextField.text else { return }
```